manifests/haproxy/binding.pp

```diff
@@ -0,0 +1,50 @@
+# Copyright (c) 2019 The Regents of the University of Michigan.
+# All Rights Reserved. Licensed according to the terms of the Revised
+# BSD License. See LICENSE.txt for details.
+
+# Load-balanced frontend
+#
+# @example
+#   nebula::haproxy_service { 'namevar': }
+define nebula::haproxy::binding(
+  String $service,
+  Boolean $https_offload = true,
+) {
+
+  $last_octet = $::ipaddress.split('\.')[-1]
+  $cookie = "cookie s${last_octet}"
+  $track = "track ${service}-${::datacenter}-https-back/${::hostname}"
+
+  # TODO - handle case where https_offload = false - ie proxy https to server
+
+  concat_fragment { "${service}-${::datacenter}-http ${::hostname} binding":
+    target  => "/etc/haproxy/services.d/${service}-http.cfg",
+    order   => '04',
```

Comment:
should look to see how order is used in "real" puppet modules

Reply:
looks fine (looking at e.g. the puppetlabs apache module); also, we're likely to migrate longer term to the puppetlabs haproxy module, so I think this is OK as is